molecule/__init__.py

```diff
@@ -17,8 +18,13 @@
 #  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
 #  FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
 #  DEALINGS IN THE SOFTWARE.
+"""Molecule version information."""
 
-import pbr.version
+from __future__ import absolute_import, division, print_function
```

Comment:
```suggestion
from __future__ import absolute_import print_function
```

?

Reply:
It's just a snippet used across Ansible. I don't want to customize it.